disnake/asset.py

```diff
@@ -48,6 +53,17 @@
 
 
 class AssetMixin:
+    """
+    Base type for all asset-like types.
+    This currently includes:
+
+    - :class:`Asset`
+    - :class:`Emoji`
+    - :class:`PartialEmoji`
+    - :class:`StickerItem`
+    - :class:`Sticker`
+    """
```

Comment:
probably can be removed? Not entirely sure

Reply:
cc3f3fc22552f4f439545ea6d7997b8a3725709d